src/applications/personalization/profile/components/proof-of-veteran-status/ProofOfVeteranStatus.jsx

```diff
@@ -66,55 +68,99 @@ const ProofOfVeteranStatus = ({
+                text="Download and print your Veteran status card"
+                onClick={createPdf}
+              />
+            </div>
+            <div className="vads-l-grid-container--full vads-u-padding-y--2">
+              <div className="vads-l-row">
+                <div className="vads-l-col--12 xsmall-screen:vads-l-col--12 small-screen:vads-l-col--7 medium-screen:vads-l-col--5 ">
+                  <img
+                    width="100%"
+                    src="/img/proof-of-veteran-status-card-sample.png"
+                    alt="sample proof of veteran status card featuring name, date of birth, disability rating and period of service"
+                  />
+                </div>
               </div>
             </div>
-          </div>
-          <div className="vads-u-font-size--md">
-            <va-link
-              download
-              filetype="PDF"
-              // exception to eslint: the url is a dynamically generated blob url
-              // eslint-disable-next-line no-script-url
-              href="javascript:void(0)"
-              text="Download Veteran status card"
-              onClick={createPdf}
-            />
-          </div>
-          <div className="vads-u-margin-y--4">
-            <MobileAppCallout
-              headingText="Get proof of Veteran status on your mobile device"
-              bodyText={
-                <>
-                  You can use our mobile app to get proof of Veteran status. To
-                  get started, download the{' '}
-                  <strong> VA: Health and Benefits </strong> mobile app.
-                </>
-              }
-            />
-          </div>
-        </div>
-      ) : null}
+            <div className="vads-u-margin-y--4">
+              <MobileAppCallout
+                headingText="Get proof of Veteran status on your mobile device"
+                bodyText={
+                  <>
+                    You can use our mobile app to get proof of Veteran status.
+                    To get started, download the{' '}
+                    <strong> VA: Health and Benefits </strong> mobile app.
+                  </>
+                }
+              />
+            </div>
+          </>
+        ) : null}
+
+        {serviceHistory.length > 0 && eligibilityIndicator === 'N' ? (
+          <va-alert
+            close-btn-aria-label="Close notification"
+            status="warning"
+            visible
+          >
+            <>
+              <p className="vads-u-margin-top--0">
+                Our records show that you’re not eligible for a Veteran status
+                card. To get a Veteran status card, you must have received an
+                honorable discharge for at least one period of service.
+              </p>
+              <p>
+                If you think your discharge status is incorrect, call the
+                Defense Manpower Data Center at 800-538-9552 (TTY 711). They’re
```

Comment:
I think we should have this phone number use the `va-telephone` component so that it is a clickable link, along with the 711 part as well and pull those from constants instead of hard coded.

``` js
<p>
      If you think your discharge status is incorrect, call the
      Defense Manpower Data Center at{' '}
      <va-telephone contact={CONTACTS.DS_LOGON} /> (
      <va-telephone contact={CONTACTS[711]} tty />
      ). They’re open Monday through Friday, 8:00 a.m. to 8:00 p.m.
      ET.
</p>
```

^ Is usually how we would do it and using the [`CONTACTS` constant that comes from the design system component](https://github.com/department-of-veterans-affairs/component-library/blob/main/packages/web-components/src/contacts.ts#L96)

Reply:
Done! Screenshot of the links in action:

<img width="657" alt="Screenshot 2024-05-22 at 3 09 26 PM" src="https://github.com/department-of-veterans-affairs/vets-website/assets/1807967/51e7c246-9abc-4113-bcd6-7b0478c1c7da">